src/measure_theory/integral/limit_comparison.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2022 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+
+import measure_theory.integral.interval_integral
+
+import analysis.special_functions.exponential
+import analysis.special_functions.integrals
+import measure_theory.integral.integral_eq_improper
+
+
+/-!
+# Limit comparison test for integrals
+
+As easy special cases of general theorems in the library, we prove the following tests
+for integrability:
+
+* If f is continuous on [a,∞), for some a ∈ ℝ, and there exists b > 0 such that
```

Comment:
Can you please add the name of the theorem please? Something like

* `name`: statement

`name` should be the full name (including the namespace), thanks!

Reply:
Fixed on the other PR.